spec/models/passing_game_log_spec.rb

```diff
@@ -5,4 +5,22 @@
   describe "associations" do 
     it { should belong_to(:player) }    
   end
+
+  describe "completion percentage" do
+    it "should calculate completion percentage rounded to 1 decimal point" do
+      passing = create(:passing_game_log)
+      CP = ((passing.completions.to_f / passing.passing_attempts.to_f) * 100).round(1)
+
+      expect(passing.completion_percentage).to eq(CP)
+    end
```

Comment:
Consider using hardcoded numbers here rather than relying on the calculations on line 12 and 21.

Reasons to favor hardcoded values over calculations in *tests*:

- If there's ever a bug in the test's calculation that is *also* introduced in the `PassingGameLog#completion_percentage` method, then the test would continue to pass when we'd want to see it fail 
- Reading the hardcoded values in the test shows exactly what is expected, e.g. `54.1`, without having to figure out what format the result of the calcuation of `CP` will take. This makes the test a little more readable.